### DISCUSSION TITLE
it is possible to add BT support for 10E version?

### DISCUSSION THREAD

**Discussion by linuxdep** [syssi/esphome-seplos-bms#51] [Ideas]
at moment my seplos battery pack is offline, but it is possible to add BT communication? The new version 10E of Seplos BMS has a BT module to read and write date to bms?
Depending on how long my rebuild lasts, I could collect data from BT communication for you if necessary.

**Comment by syssi** [comment]
I would be happy about a btsnoop capture. Do you need some guidance?

**Reply by linuxdep** [reply]
yes i have see this in a other thread... but you can add a link please.

**Reply by syssi** [reply]
Please try to follow this guide: https://medium.com/@charlie.d.anderson/how-to-get-the-bluetooth-host-controller-interface-logs-from-a-modern-android-phone-d23bde00b9fa

**Reply by Meli73** [reply]
An update to this How to: You don't need any PC tools to get the bugreport anymore. After enabling developer mode the Google Files App has access to bugreports and you could easily copy them e.g. to google drive ;-) at least with android 13

**Reply by linuxdep** [reply]
thanks for this info... at weekend i test it

**Reply by linuxdep** [reply]
I am unfortunately ill, will drag on a bit, too bad.

**Reply by zekeboz** [reply]
@linuxdep any luck in polling the 10e via BT? Thanks!

**Comment by NicoThien** [comment]
ich hab mal probiert eine Komunikation aufzuzeichnen. Hab aber keine Ahnung ob was sinnvolles dabei ist.
[btsnoop_hci.log](https://github.com/syssi/esphome-seplos-bms/files/11804697/btsnoop_hci.log)
Ich hab zwei 10E Seplos BMS

**Comment by Mechan2** [comment]
Any updates on this?

**Comment by Vermond1** [comment]
So far, I have been unable to get any Syssi Seplos-Bms-yaml through my Esp32-RS485 logger to get data from my Manyi battery which contains a Seplos Bms version 2.0 clone. Connecting goes fine, Tx and Rx LEDs flash as if there is data transport. However, no data is reported back, very frustrating. Is the version 3 different? Is there a version 2.0 yaml available?

**Comment by syssi** [comment]
This is the BLE component to talk to a Seplos BMS v2: https://github.com/syssi/esphome-seplos-bms/blob/main/esp32-ble-example.yaml
